hivemq-edge/src/test/resources/configs/simulation/simulation-adapter-full-config.xml

```diff
@@ -0,0 +1,43 @@
+<hivemq>
+    <protocol-adapters>
+        <simulation>
+            <id>my-simulation-protocol-adapter</id>
+            <minValue>0</minValue>
+            <maxValue>1000</maxValue>
+            <minDelay>0</minDelay>
+            <maxDelay>1000</maxDelay>
+            <simulationToMqtt>
+                <pollingIntervalMillis>10</pollingIntervalMillis>
+                <maxPollingErrorsBeforeRemoval>9</maxPollingErrorsBeforeRemoval>
+                <simulationToMqttMappings>
+                    <simulationToMqttMapping>
+                        <mqttTopic>my/topic</mqttTopic>
+                        <mqttQos>1</mqttQos>
+                        <messageHandlingOptions>MQTTMessagePerTag</messageHandlingOptions>
```

Comment:
Maybe we should change this to MQTTMessagePerSubscription, as MQTTMessagePerTag is the default.

Reply:
i dont understand what these options do anyway, so hey feel free